fracatal/scripts.py

```diff
@@ -52,7 +54,11 @@ def v_stability_sweep(pattern, make_kernel, my_update, \
     dim_h = int(native_dim_h * scale_factor)
     dim_w = int(native_dim_w * scale_factor)
     
-    scaled_pattern = np.array(skimage.transform.resize(pattern, (1,1, dim_h, dim_w)), dtype=default_dtype)
+    if scale_factor < 1.0:
```

Comment:
Using a high order interpolation seems to remove the peculiar speedup (probably related to JIT compilation avoiding redundant computation).